packages/css/src/form/fieldset/fieldset.css

```diff
@@ -6,6 +6,7 @@
 .hds-fieldset {
   display: flex;
   flex-flow: column nowrap;
+  gap: var(--hds-spacing-4);
```

Comment:
Will this be correct? From figma it's 16 once place
<img width="385" alt="image" src="https://github.com/bring/hedwig-design-system/assets/244257/34319b81-1af0-4b32-b3a7-0a18da27454d">

But then somewhere else 8
<img width="458" alt="image" src="https://github.com/bring/hedwig-design-system/assets/244257/f591cfae-216d-40d8-9a39-f27d4dfec896">


Or am I looking at the wrong reference?

Reply:
You're right. I asked Morten, and he said different content might need different spacing.
8 for smaller things like radiobuttons and checkboxes
16 for things like input fields

We agreed 8 is a good default, but the user must be able to easily override this. I'll change the default to 8, and look into the overriding later. I have a feeling it might be up to import order to "randomly" decide specificity here if user tries to override.

Also, I'll move this to a separate PR, so we can get it merged, as this PR will change more based on the things we talked about regarding the checkbox and fieldset.

Reply:
You find this now in #186